### PR TITLE
Create gl.yml

### PR DESCRIPTION
**Pull request by D3KRISH**
Galician language yml for Team ultroid from D3VILKRISH

### CODE REVIEW

File: strings/strings/gl.yml [hunk 1]
```diff
@@ -0,0 +1,594 @@
...
+help_9: "Configuración: engrenaxe:"
+help_10: "•• Cʟᴏꜱᴇ ••"
+help_11: "Nome do complemento -{} \ n \ n✘ comandos dispoñibles -\ n \ n"
+help_12: "A chamada de voz non comeza"
+help_13: "Vostede fai '{} setDb Addons True" para que os complementos poidan vir "
+
+# userlogs
+userlogs_1: "O ID de chat que introduciu é incorrecto, corríxelo"
+userlogs_2: "Póñame a etiqueta a xente en grupo para que a xente poida ser"
+userlogs_3: "Chat" "
```
Comment (New-dev0) [inline strings/strings/gl.yml:41]: Invalid string line

File: strings/strings/gl.yml [hunk 2]
```diff
@@ -0,0 +1,594 @@
...
+adm_3: "Dime o tempo ..."
+pro_1: "promove respondendo a unha persoa"
+pro_2: "{} é agora o administrador {} co nome {}"
+de_1: "DeMote respondendo a unha persoa"
+de_2: "{} xa non no administrador {}"
+ban_1: "Ban o ID do usuario ou o nome de usuario"
+ban_2: "Non podo prohibir ao meu fabricante"
+ban_3: "Non teño dereito a prohibir ao usuario."
+ban_4: "{} {} prohibido {} en {}"
+ban_5: "\ n Reason: {}"
```
Comment (New-dev0) [inline strings/strings/gl.yml:57]: Seems like you used some kind of translator.
Fix \ n and \ N present in file..

File: strings/strings/gl.yml [hunk 3]
```diff
@@ -0,0 +1,594 @@
...
+Autopic_1: "Dame algunhas frases de asunto ..."
+Autopic_2: "Non hai resultados atopados para" {} "
+Autopic_3: "{} atopou algúns resultados relacionados co teu descubrimento! \ N Autopic comezou!"
+Autopic_4: "Non se está a usar o autópico !!"
+Autopic_5: "Autópico pechado!"
+
+# EN LIÑA
+inline_1: "[soporte vomitroid] (t.me/ultroidsupport) \ n \ n \ n \ n {} Menú de helpa. \ N \ n \ n \ n \ n Plug-in ~ {}"
+inline_2: "[soporte vomitroid] (t.me/ultroidsupport) \ n \ n \ n \ n {} Menú de helpa. \ N \ n \ n \ n \ n adon ~ {}"
+inline_3: "[soporte vomitroid] (t.me/ultroidsupport) \ n \ n \ n {} का मदद मेनू. \ n \ n \ n एडऑन मूल daथ \ n \ n ज.
```
Comment (New-dev0) [inline strings/strings/gl.yml:122]: Missing closing end

File: strings/strings/gl.yml [hunk 4]
```diff
@@ -0,0 +1,594 @@
...
+#chats
+chats_1: "O proceso continúa ......"
+chats_2: "Este chat non se pode eliminar"
+chats_3: "#deleted \ n eliminado {}"
+chats_4: "O teu [{}] ({}) converteuse en grupo!"
+chats_5: "Únete a @Teamultroid"
+chats_6: "O teu xefe de grupo / canle creouse!"
+
+# Acción limpa
+clan _1: "Engadido configuración de acción limpa para este chat"
```
Comment (New-dev0) [inline strings/strings/gl.yml:246]: Gap between 'clan' & '_1'

**Review by New-dev0 — CHANGES_REQUESTED** [review]

File: strings/strings/gl.yml (hunk 2 above)
Comment (D3KRISH) [reply strings/strings/gl.yml:57]: Yes

File: strings/strings/gl.yml (hunk 3 above)
Comment (D3KRISH) [reply strings/strings/gl.yml:122]: Sorry bro I forgot to closing this

**Comment by New-dev0** [comment]
Commit the mentioned changes and create new pull request targeting main branch

**Pull request closed without merging**